packages/server/src/alerts/alerts.controller.ts

```diff
@@ -33,16 +32,13 @@ export class AlertsController {
     @Param('courseId') courseId: number,
     @User() user: UserModel,
   ): Promise<GetAlertsResponse> {
-    const alerts = (
-      await AlertModel.find({
-        where: {
-          courseId,
-          user,
-          resolved: null,
-        },
-      })
-    ).map((alert) => pick(alert, ['sent', 'alertType', 'payload', 'id']));
-
+    const alerts = await AlertModel.find({
+      where: {
+        courseId,
+        user,
+        resolved: null,
+      },
+    });
     return { alerts: await this.alertsService.removeStaleAlerts(alerts) };
```

Comment:
Not related to your PR exactly: but do we clean up alerts at night? I wonder what happens when when I ask a student to rephrase their question today, they never address it, and they open up the app tomorrow?

Maybe nothing because the questions are cleaned from the queue (and hence there will be no alerts in the ListQuestionsResponse). Do we want to mark alerts as Stale when closing these questions?

Reply:
i think it should be fine because theoretically this `removeStaleAlerts` gets called whenever alerts are fetched. So when the student checks in the next time in the course,  _i think_ the old alerts should be culled

```
    const alerts = await AlertModel.find({
      where: {
        courseId,
        user,
        resolved: null,
      },
    });
```